images/scripts/debug-wrapper.sh

```diff
@@ -4,9 +4,13 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+DEBUG_PORT="${DEBUG_PORT:=2345}"
+DEBUG_CONTINUE="${DEBUG_CONTINUE:=false}"
+
 /usr/bin/dlv \
-    --listen=:2345 \
+    --listen=":${DEBUG_PORT}" \
     --headless=true \
+    --continue="${DEBUG_CONTINUE}" \
```

Comment:
I like the continue step. I wonder whether it's more useful to developers to turn this on by default or off by default? For the `kind-image` target it's a bit annoying to have to go through and release each debugger any time you restart the agent or operator pods, so maybe the default should be to continue and we should instead do a `DEBUG_HOLD` var?

(maybe worth raising this in #development on Slack to see if other devs have alternate opinions on this front)

Reply:
personally, i would prefer the same. 

but i wanted to introduce this change without changing the existing logic. which is also the reason why i decided to keep `make kind-debug-agent` for the time being.

an option would be to set `DEBUG_CONTINUE` only in case of the newly introduced make target `kind-debug` and keep it as is in `kind-debug-agent`

Reply:
I suggest that we pick a proposal (I would suggest continue by default), and use the Slack and/or community meeting to share that proposal so that anyone who cares can voice their thoughts on this. It would also help others to know about (a) this debug feature overall and (b) that the behaviour will change. I had introduced the change originally and I'm on board with changing it. In this case since we're talking about changing behaviour for developers, I don't think we need to be shy about changing the behaviour.

I would prefer if we don't conditionally change the behaviour depending on the target. That will make it more confusing for anyone new who attempts to use one of the targets and switches between the targets for some reason. Let's make it consistent and pick a default that works well for 90% of cases, then allow developers to override it if they have a special case they want for their own workflow. IMO setting continue by default is the right choice here.

Reply:
:+1: 

going to 
* switch the default to `continue=true`
* rename the env variable for changing the default to `DEBUG_HOLD`
* keep the targets `kind-debug` &  `kind-debug-agent` the same regarding this default behaviour

will bring this up in slack/community meeting next week